uk.co.compucorp.civicrm.hrcore/CRM/HRCore/SearchTask/ContactFormSearchTaskAdder.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use CRM_HRCore_Form_CreateUserRecordTaskForm as CreateUserRecordTaskForm;
+use CRM_HRCore_Form_SendInvitationEmailTaskForm as SendInvitationEmailTaskForm;
+use CRM_HRCore_SearchTask_SearchTaskAdderInterface as SearchTaskAdderInterface;
+
+class CRM_HRCore_SearchTask_ContactFormSearchTaskAdder implements SearchTaskAdderInterface {
+
+  /**
+   * @param $tasks
+   */
```

Comment:
When the child methods have the same description as their parents, you can use `@inheritdoc`